src/Jira/Api.php

```diff
@@ -68,27 +68,34 @@ class Api
 	protected $options = self::AUTOMAP_FIELDS;
 
 	/**
-	 * Fields.
+	 * Client-side cache of Fields. List of fields when loaded, null when nothing is fetched yet.
 	 *
-	 * @var array
+	 * @var array|null
 	 */
 	protected $fields;
 
 	/**
-	 * Priorities.
+	 * Client-side cache of Priorities. List of priorities when loaded, null when nothing is fetched yet.
 	 *
-	 * @var array
+	 * @var array|null
 	 */
 	protected $priorities;
 
 	/**
-	 * Statuses.
+	 * Client-side cache of Statuses. List of statuses when loaded, null when nothing is fetched yet.
 	 *
-	 * @var array
+	 * @var array|null
 	 */
 	protected $statuses;
 
 	/**
+	 * Client-side cache of Resolutions. List of resolutions when loaded, null when nothing is fetched yet.
```

Comment:
The `Resolutions` should be in lowercase. It was capitalized because it was 1st word in sentence.